docs/index.md

```
@@ -0,0 +1,50 @@
+
+The ontology-based rare disease common data model (RD-CDM) is a non-balloted
+extension of the ERDRI-CDS which aims to provide an internationally viable
+template for the development and implementation of country-specific rare disease
+common data models. The RD-CDM is designed to be compatible with the GA4GH
+Phenopacket Schema, HL7 FHIR, and the International Patient Summary. The
+ontology-based approach of the RD-CDM allows for the integration of various data
+sources and the harmonization of data across different systems.
+
+!!! tip
+    The RD-CDM paper has now been published at Nature Scientific Data. You can
+    read it [here](https://www.nature.com/articles/s41597-025-04558-z)!
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)